airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/transform_models/AddFieldTransform.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.protocol.models.transform_models;
+
+import io.airbyte.protocol.models.JsonSchemaType;
+import java.util.List;
+import java.util.Objects;
+
+public class AddFieldTransform {
```

Comment:
Nit:  Add public JavaDoc to save us from having to add one when we enable PMD/Checkstyle ;)

Reply:
done!